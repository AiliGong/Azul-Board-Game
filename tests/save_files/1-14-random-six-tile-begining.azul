#myformat
3
6
0
a
b
RRBUBLBUYLBROOUROLRLOYBUROBUOBYYRLRRYLYYBUUUYYUYLOBLOYLLRURLYRLUYOUBRBORBBBLLUYOYYLOBUUOLRRYUBLYYOLBOUOBUBOYROOLOBLURRUR
