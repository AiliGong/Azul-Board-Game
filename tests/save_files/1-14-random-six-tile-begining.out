Factories:
0: F
1: R R B U
2: B L B U
3: Y L B R
4: O O U R
5: O L R L

Score for a: 0
Mosaic for a:
1:           . || . . . . . . 
2:         . . || . . . . . . 
3:       . . . || . . . . . . 
4:     . . . . || . . . . . . 
5:   . . . . . || . . . . . . 
6: . . . . . . || . . . . . . 
broken:

Score for b: 0
Mosaic for b:
1:           . || . . . . . . 
2:         . . || . . . . . . 
3:       . . . || . . . . . . 
4:     . . . . || . . . . . . 
5:   . . . . . || . . . . . . 
6: . . . . . . || . . . . . . 
broken:
